system/application/plugins/tensor/system/application/config/config.php

```diff
@@ -0,0 +1,371 @@
+<?php  if ( ! defined('BASEPATH')) exit('No direct script access allowed');
+
+/*
+|--------------------------------------------------------------------------
+| Base Site URL
```

Comment:
@arthurian - I'd like to make sure this file doesn't conflict with scalar's config in a way that I'm not considering.